apiserver/facades/client/application/get_test.go

```diff
@@ -101,6 +101,65 @@ func (s *getSuite) TestClientApplicationGetSmoketestV5(c *gc.C) {
 	})
 }
 
+func (s *getSuite) TestClientApplicationGetSmoketestV6(c *gc.C) {
```

Comment:
We don't need V6 and V7 tests here. The change in functionality for Get() was in V5. V6 abd V7 will use V5 Get()